src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import './index.scss';
+import App from './App/App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
Could we remove the `serviceWorker` ? I am thinking that it might confuse some junior devs no?